build_tools/ci/cpu_comparison/run.py

```diff
@@ -1707,7 +1719,9 @@ def __init__(self):
                 test_params=TestParams(
                     use_ukernel=True,
                     use_chess=False,
```

Comment:
nit: This can be dropped.

Reply:
I don't think it hurts to be explicit about it, and I also didn't touch this line.